packages/app-builder/src/components/ui/user/create-page.tsx

```diff
@@ -0,0 +1,158 @@
+import { NodeTree, SerializedNode, useEditor, Element } from '@craftjs/core'
+import { serializeNode } from '../../../utils/serializeNode'
+import { Button } from '@reapit/elements'
+import { newPage } from '../header/PageSelector'
+import { useUpdatePage } from '@/components/hooks/apps/use-update-app'
+import { nodesObjtoToArr } from '@/components/hooks/apps/node-helpers'
+import { usePageId } from '@/components/hooks/use-page-id'
+import { useApp } from '@/components/hooks/apps/use-app'
+import { emptyState } from '@/components/hooks/apps/emptyState'
+import { useObjectMutate } from '@/components/hooks/objects/use-object-mutate'
+import { ParsedArg } from '@/components/hooks/use-introspection/query-generators'
+import { FormInput } from './form-input'
+import Form from './form'
+import { resolver } from '@/components/pages/home'
+import React from 'react'
+import Container from './container'
+import Text from './text'
+
+const constructPageNodes = (
+  typeName: string,
+  operationType: string,
+  reactElementToNodeTree: (element: React.ReactElement) => NodeTree,
+  args?: ParsedArg[],
+  sourcePageId?: string,
+  pageTitle?: string,
+): Record<string, SerializedNode> => {
+  const { ROOT } = emptyState
+
+  const nodes = {}
+
+  if (operationType === 'list') {
+    // table
+    nodes['743hfu'] = {
```

Comment:
would it be handy at all if `743hfu` was a constant? Is there any significant of this value?